warehouse/static/sass/blocks/_notification-bar.scss

```diff
@@ -33,6 +33,30 @@
     - dismissable: Indicates notification be be dismissed. Defaults to hidden.
     - visible: Indicates a visible, non-dismissed notification
 */
+/*
+  A notification bar can be made dismissable by adding the following to the main
+  notification-bar div:
+    - .notification-bar--dismissable to it's classes
```

Comment:
"it's" should be "its" https://data.grammarbook.com/blog/pronouns/1-grammar-error/